src/flag_gems/ops/masked_fill.py

```diff
@@ -33,8 +34,8 @@ def masked_fill_kernel(inp, expand_mask, value, out, N, BLOCK_SIZE: tl.constexpr
 @libentry()
 @triton.autotune(configs=cfggen(), key=["N"])
 @triton.jit
-def masked_fill_kernel_self(inp, expand_mask, value, N, BLOCK_SIZE: tl.constexpr):
-    pid = tl.program_id(axis=0)
+def masked_fill_kernel_self(inp, expand_mask, value, out, N, BLOCK_SIZE: tl.constexpr):

```

Comment:
argument out was added by mistake. please remove it and submit a bug fix pr.

Reply:
I checkout from master from before masked_fill is merged, where there used to be an out parameter.